Engine/csound_orc_compile.c

```diff
@@ -839,7 +779,8 @@ INSTRTXT *create_instrument0(CSOUND *csound, TREE *root,
 
   if (csound->ksmps != DFLT_KSMPS) {
     reallocateVarPoolMemory(csound, engineState->varPool);
-    // csound->Message(csound, "recalculate varpool\n");
+    if(csound->GetDebug(csound))
```

Comment:
Sidenote: this seems like a common pattern, we might benefit from a macro or function such as `debugMessage(csound, "some string")`